microsoft/atlas-design
# Fix form behavior error summary duplicates

Task: task-[662148](https://dev.azure.com/ceapex/Engineering/_workitems/edit/662148/), task-[655260](https://dev.azure.com/ceapex/Engineering/_workitems/edit/655260/)

Link: preview-[421](https://design.docs.microsoft.com/pulls/421/components/form.html)

This PR aims to achieve the following:
- Remove duplicate messages in the form behavior error summary for Firefox.
- Adjust the scroll margin for field body content elements.
- Adds focus to star rating as custom elements do not have focus by default.

## Testing

1. Visit https://design.docs.microsoft.com/pulls/421/components/form.html with Firefox. Scroll down to form behavior section. Click on its Submit button.
2. The error summary should be visible and it should not contain duplicate messages. Check with other browsers.
![image](https://user-images.githubusercontent.com/94572161/179863163-802e2e74-dd92-4f93-a2ef-bbd1ea4a6889.png)
3. Click on "A selection for "Star rating" is required" error message link.  The window should jump to the star rating and its label should be in view after the jump. The first star should have focus outline.
![image](https://user-images.githubusercontent.com/94572161/180509385-3617b3e1-8136-4a61-a8ab-d59c56a126a1.png)
